src/azure-cli/azure/cli/command_modules/storage/_transformers.py

```diff
@@ -130,4 +130,17 @@ def transform_url(result):
     result = re.sub('/', '//', result, count=1)
     return encode_url_path(result)
 
+
+def transform_storage_table_policy_list_output(result):
+    new_result = []
+    if result is None:
+        return new_result
+    for policy_name in result.keys():
+        new_entry = {'name': policy_name}
```

Comment:
It looks that you changed output structure compared to previous format, which may cause breaking change to users and is not expected. May I know the reason why you have such design? and can we consider not changing output format only with [] change?

Reply:
Yes, this is a breaking change.
If only change the outermost `{}` to `[]`, the returned one will not be in the correct JSON format
If only `[]` is added to the outermost layer, it is also breaking change, and this format is not really returned as a list
My return format is the one recommended by tjprescott in the issue, he means
> This would be consistent with the rest of Azure and thus the user could employ the same patterns for searching for the desired object.